Use REQUIRE so eautopoint --force will upgrade gettext to 0.22 if available.

--- a/configure.ac
+++ b/configure.ac
@@ -131,7 +131,7 @@ ALL_LINGUAS="cs da de es fr fi hu id ja ka ko nb pl pt_BR ro ru rw sr sv vi zh_C
 
 dnl Checks for programs.
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.21])
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.21])
 
 dnl This is a hacky workaround as the m4 scripts for iconv seem to set
 dnl LIBICONV to '-liconv' despite that this lib is not available if iconv
